apps/activity_watcher.py

```diff
@@ -94,7 +94,7 @@ def process_message(message):
 
         # source events
         if context[u'topic'] == 'source' and (body[u'type'] == 'insert' or body[u'type'] == 'modify') and \
-                (body[u'source'] == '6' or body[u'source'] == '6'):
+                (body[u'source'] in pac.conf['SEND_PAC_SOURCES']):
```

Comment:
I have some concerns about a blank or null pac.conf value.

Reply:
Yes. I share those concerns, as well as with the settings are being loaded [out of scope, here]. 
If you wanted to address the null/empty values in this PR, or at least for certain keys, you can use the `.get()` method of the conf dict instead of the `[]` syntax; that would allow you to provide a default and/or handle a null value:

```python
from typing import Any


def get_setting(key: str, allow_none: bool = False, allow_empty: bool = False) -> Any:
    value = pac.conf.get('key')
    if value is None and not allow_none:
        raise KeyError(key)
    elif not value and not allow_empty:
        raise ValueError(f'Expected {key} to have a value, but it was {value}')
   return value


# . . . 

def do_work():
    smtp_conn_type = get_setting('SMTP_CONN_TYPE')  # Will require the value by default
```